source/guides/first-steps/task-script.md

```diff
@@ -1,4 +1,121 @@
+
+Previously, you've learned how to create an environment in VS Code, the editor that we recommend for writing Denizen scripts.
+
+#### Creating the file
+
+To create your first task script, start by opening your scripts folder in VS Code.
+
+TODO SCREENSHOT (open scripts folder)
+
+From there, right click the scripts folder in the explorer menu and click the "New File" option.
+
+TODO SCREENSHOT (new file highlighted)
+
+Once your file has been created, you can name it something like `MyTask.dsc`. Make sure to use the `.dsc` extension so VS Code recognizes it as a Denizen script, but the *name* of the file can be whatever you like. Usually, it's helpful to name the file something relevant to the script(s) that it contains.
+
+From here, double-click the file to open it in VS Code.
+
+TODO SCREENSHOT (opened file)
+
+Now you can begin writing your first task script!
+
+#### Writing the script
+
+Let's start with the core of the script:
+
+```dscript_blue
+my_first_task:
+    type: task
+    script:
+    - narrate (sometext)
+```
+
+This example should look familiar - it's very similar to the example above.
+
+### Script names
+
+The name of the script here is `myfirsttask`. It's at the top of the indentation. Each top-level indentation entry is a separate script. Here is an example of two different scripts:
+
+```dscript_green
+myfirsttask:
+    type: task
+    script:
+    - narrate "This is is task number one!"
+
+mysecondtask:
+    type: task
+    script:
+    - narratee "This is task number two!"
+```
+
+This example demonstrates what having two different scripts in the same file looks like. `myfirsttask` and `mysecondtask` are two different scripts, and each is a fully self-contained script.
+
+### Script types
+
+Below the script name - on the second level of indentation - you'll see the `type` and `script` entries.
+
+The `type` option is where you specify which `type` of Denizen script you're writing. In this case, we're writing a `task` script. You may have seen `world` scripts, `item` scripts, `inventory` scripts, and *maybe* even a `procedure` script. For now, let's focus on `task` scripts. Make sure you write `type: task` in your script on the second level of indentation, below the script name, like this:
+
+```dscript_blue
+myfirsttask:
+    type: task
+```
+
+### Script Commands
+
+Below the `type` entry, you'll see the `script` entry. This is where you'll be doing most of your work in Denizen! Under the `script` entry is where you write the content of the script - no matter which kind of script it is, the `script` entry is where you tell Denizen what to do. There are many different script types - a few of them being world scripts, assignment scripts, command scripts, and item scripts. You'll learn about the different types as you read through these guides, and you can find explanations about all of them from the meta documentation on the [language explanation page](https://one.denizenscript.com/denizen/lngs/).
```

Comment:
`no matter which kind of script it is, the 'script' entry is where you tell Denizen what to do.`
This is false ^
Consider for example world scripts. They do not have a `script` key. Only `task`, `procedure`, and `command` scripts use `script`.

Reply:
The discussion about many script types should be moved into the section above (where the pre-existing comment about adding that is placed).

Also, the `/lngs` link should probably be changed to `https://one.denizenscript.com/denizen/lngs/container` as a more specific direct link.